### ISSUE TITLE
Unwanted string `syntax=` appears when in insert mode.

### ISSUE THREAD

**Issue by mattfinnell** [airblade/vim-gitgutter#430]
![vimsyntaxerror](https://user-images.githubusercontent.com/4606321/28601623-28a5d034-716e-11e7-96eb-d12eabbf1abf.png)

Title should state most. This non-deterministically occurs when entering insert mode.

**Comment by airblade** [comment]
How odd.  Are you sure this is due to vim-gitgutter?  I can't think of anything that would cause this, and `syntax` doesn't appear in the code anywhere.

**Comment by airblade** [comment]
I'm confident this isn't caused by vim-gitgutter so I'm closing it.  Happy to reopen if you can show otherwise.

**Comment by mattfinnell** [comment]
So I recently rebuilt my vim configuration and reincorporated `vim-gitgutter`. I haven't come across any problems with the stray string and am now loving your creation!

For some more slightly more useful information. I swapped out `pathogen` for `plug`. I think due to `plug`'s lazy evaluation, all the plugins are now being loaded in a nice order in a minimal manner. Where pathogen might have been loading all plugins at once without planning load-order.
